sources/packages/web/src/views/institution/locations/programs/LocationProgramAddEdit.vue

```diff
@@ -1,30 +1,40 @@
 <template>
-  <h5 class="text-muted">
-    <span v-if="programId">Edit Program</span>
-    <span v-else>Create New Program</span>
-  </h5>
-  <v-sheet elevation="1" class="mx-auto">
-    <v-container>
-      <formio
-        formName="educationprogram"
-        :data="initialData"
-        @submitted="submitted"
-      ></formio>
-    </v-container>
-  </v-sheet>
+  <p class="muted-heading-text">
+    <a @click="goBack()">
+      <v-icon left> mdi-arrow-left </v-icon> Program detail</a
+    >
+  </p>
+  <span class="heading-x-large">
+    <span v-if="isReadonly">View Program</span>
+    <span v-if="programId && !isReadonly">Edit Program</span>
```

Comment:
why is programId checked here? can it be null for this component?

Reply:
yes, as in the label, if `programId` is not there, then, it is in create mode